Tests/SwiftStackTests/SiteTests.swift

```diff
@@ -20,4 +22,26 @@ class SiteTests: XCTestCase {
         XCTAssertNotNil(jsonSite)
     }
     
+    
+    func testFetchSites() {
```

Comment:
When unit testing, let's not perform actual API requests.  `TestableClient` has a function that allows us to intercept requests and return a "fake" response; we'll use that to test requests.  Making real API calls is slow and it uses up API quota.